src/biogeophys/HumanIndexMod.F90

```diff
@@ -1014,6 +1014,15 @@ subroutine Wet_BulbS (Tc_6,rh,wbt)
 ! !LOCAL VARIABLES:
 !EOP
 !
+#ifndef NDEBUG
```

Comment:
Do we really want this #ifdef here?

Reply:
This was on purpose. But, I'm not sure if really should be done.

Was it expensive?

Reply:
This was on purpose. But, I'm not sure if really should be done (the #if...)

Was it expensive? No not more expensive than the expression below which is expensive, and can blow up for bad values.

And this WetBulb stuff is already known to be expensive, hence we don't turn it on that often.

So final bottom-line remove the #ifndef and the #endif

Reply:
Gone!